### PR TITLE
Removing deprecated undocumented syntax for binding a scope to several types

### PR DESCRIPTION
**Pull request by herbelin**
**Kind:** removal of undocumented deprecated syntax

This is a following of #7762, which it includes, targetting _8.10_.

- [X] Entry added in CHANGES.

### CODE REVIEW

File: vernac/g_vernac.mlg [hunk 1]
```diff
@@ -1119,7 +1119,8 @@ GRAMMAR EXTEND Gram
 	 { VernacDelimiters (sc, None) }
 
      | IDENT "Bind"; IDENT "Scope"; sc = IDENT; "with";
-       refl = LIST1 class_rawexpr -> { VernacBindScope (sc,refl) }
+       refl = class_rawexpr ->
+         { VernacBindScope (sc,[refl]) }
```
[the remark this answers is not in the document]
Comment (herbelin) [reply vernac/g_vernac.mlg]: Laziness... and the burden of changing the API, since it causes no harm and the inner code supports anyway multiple types.

**Comment by herbelin** [comment]
Well, if the syntax is actually known in the wild, maybe I made a wrong choice by removing it. Another way to answer #7699 is also to say that it is not a bug.

**Comment by Zimmi48** [comment]
A PR on VST can be an opportunity to ask the authors what they think of this syntax and its potential removal.

**Comment by Zimmi48** [comment]
@herbelin I let you decide whether you want to prepare a PR on VST or close this (and #7762 at the same time). In the meantime, this is going to be off my radar.

**Comment by herbelin** [comment]
@zimmi48: I would let it open for a while if ever some user want to comment. In the absence of comment, it shall probably rather be correcting the documentation.

**Pull request closed without merging**